src/main/app/claims/offerClient.ts

```diff
@@ -4,42 +4,45 @@ import { Claim } from 'claims/models/claim'
 import { User } from 'idam/user'
 import { Offer as OfferForm } from 'features/offer/form/models/offer'
 import * as config from 'config'
-import { request } from 'client/request'
+import { request as requestPromiseApi, RequestPromiseAPI } from 'client/request'
 
 export const claimStoreApiUrl: string = `${config.get<string>('claim-store.url')}/claims`
 
 export class OfferClient {
+  constructor (private request: RequestPromiseAPI = requestPromiseApi) {
```

Comment:
Why do you need to do this? I can't see any places where you initialise the client with something else.